src/pages/billing/NewBillingProjectWizard/StepWizard/Step.ts

```diff
@@ -18,16 +18,15 @@ function stepBanner(active: boolean): CSSProperties {
   }
 }
 
-export interface StepProps {
+export type StepProps = PropsWithChildren<{
   isActive: boolean
   style?: React.CSSProperties
-  children?: React.ReactNode[]
-}
+}>
 
-export const Step = ({ isActive, children = [], ...props }: StepProps) => li({
+export const Step = ({ isActive, children, style }: StepProps) => li({
```

Comment:
Since only one property of `...props` was used, it's clearer to fully destructure props. Generally, `...props` is useful for passing through most props to another component.

Reply:
Also, `children` should default to undefined instead of an empty array.